README.markdown

```diff
@@ -614,6 +615,23 @@ let value = numbers
     .filter {$0 > 50}
     .map {$0 + 10}
 ```
+## Attributes
+When specifying an attribute such as `@discardableResult` or `@available` for a declaration or type, it should be placed in a new line above the declaration. This helps functions from getting too long for a single line.
+
+**Preferred**:
+```swift
+@discardableResult
+func sum(a: Int, b: Int) -> Int {
+    return a + b
+}
+```
+
+**Not Preferred**:
+```swift
+@discardableResult func sum(a: Int, b: Int) -> Int {
+    return a + b
+}
+```
```

Comment:
Either way works for me. If the main reason to put the attribute on a separate line is to avoid long function declaration lines then having an exception for `@objc` seems fine since it usually won't make a function declaration too long. But if we want to have a single style for all attributes regardless of line length, then that seems fine too

Reply:
@onuniverse/ios-team ?

Reply:
I'm supportive of either. I guess I lean toward just doing it all the time so the style guide is easier to understand and enforce without exceptions.

Reply:
I like the idea of a consistent style for all attributes.

Reply:
My first thought was "does this also apply to `@objc`?" But consistency might be better. Kinda like when we decided that guard returns should always be on a new line: we discussed making an exception when unwrapping a weak `self`, but ultimately landed on a hard rule